http-client/src/main/java/com/facebook/airlift/http/client/thrift/ThriftRequestUtils.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.airlift.http.client.thrift;
+
+import com.facebook.airlift.http.client.Request;
+import com.facebook.drift.codec.ThriftCodec;
+
+import static com.facebook.airlift.http.client.Request.Builder.prepareGet;
+import static com.facebook.airlift.http.client.Request.Builder.preparePost;
+import static com.google.common.net.HttpHeaders.ACCEPT;
+import static com.google.common.net.HttpHeaders.CONTENT_TYPE;
+
+public class ThriftRequestUtils
+{
+    private ThriftRequestUtils() {}
+
+    public static Request.Builder prepareThriftPost(ThriftProtocol thriftProtocol)
+    {
+        String type = "application/x-thrift; t=" + thriftProtocol.name();
```

Comment:
I would suggest using the `MediaType` class and its `get/setAttribute` methods to generate the proper media type header

Reply:
Also we might want to lowercase the name (the content type header could be case insensitive)